style.css

```diff
@@ -172,6 +172,24 @@ body {
     align-items: center;
     justify-content: center;
   }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/app.creation/pullRequest?prid=6294888)